test/test_other.py

```diff
@@ -13374,7 +13374,7 @@ def test_split_module(self, customLoader, jspi, opt):
       self.emcc_args += ['-g', '-sJSPI_EXPORTS=say_hello']
     self.emcc_args += ['-sEXPORTED_FUNCTIONS=_malloc,_free']
     output = self.do_other_test('test_split_module.c')
-    if jspi:
```

Comment:
Does node canary support JSPI yet? I guess maybe with a flag, and soon without? Or does it need changes in node in addition to its V8?

Reply:
This condition here is not related to JSPI per sey, but more about the fact that d8 cannot write files out to disc.

The `if jspi:` condition was really trying to say `if d8:` and it worked since we only ever ran JSPI tests under d8 I guess.   Running this test with node canary ended up in  situation where were both using node and jspi which broke this check.